src/validation.cpp

```diff
@@ -1090,6 +1090,33 @@ bool AcceptToMemoryPool(CTxMemPool& pool, TxValidationState &state, const CTrans
     return AcceptToMemoryPoolWithTime(chainparams, pool, state, tx, GetTime(), plTxnReplaced, bypass_limits, test_accept, fee_out);
 }
 
+// Calculate the fee of a transaction by looking up inputs from the mempool if
```

Comment:
This would be better as a doxygen comment next to the declaration in the header file.